src/utils/svgHelper.ts

```diff
@@ -0,0 +1,82 @@
+/**
+ * -------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.
+ * See License in the project root for license information.
+ * -------------------------------------------------------------------------------------------
+ */
+import { html } from 'lit-element';
+export function getSVG(svg: string, color: string) {
```

Comment:
change svg to enumeration, makes it easier to use and avoid mistakes. Also use switch and case instead of else if

Reply:
updated here, enum is defined in mgt-person-card: 30398bb